src/JHipsterNetSampleApplication/Data/ApplicationDatabaseContext.cs

```diff
@@ -43,6 +44,17 @@ protected override void OnModelCreating(ModelBuilder builder)
                 .HasForeignKey(e => e.UserId)
                 .IsRequired()
                 .OnDelete(DeleteBehavior.Cascade);
+
+            builder.Entity<OperationLabel>()
+                .HasKey(t => new { t.OperationId, t.LabelId });
+
+            builder.Entity<OperationLabel>()
+                .HasOne(pt => pt.Operation)
+                .WithMany("OperationLabels");
```

Comment:
If possible I would rather use the lambda/class notation because you have a compiler (and visible IDE) error if you have a typo. String are too tricky.